airflow/contrib/hooks/neo4j_hook.py

```diff
@@ -0,0 +1,101 @@
+# -*- coding: utf-8 -*-
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+"""This hook provides minimal thin wrapper around the neo4j python library to provide query execution"""
+from neo4j import BoltStatementResult, Driver, GraphDatabase, Session
+
+from airflow.hooks.base_hook import BaseHook
+
+
+class Neo4JHook(BaseHook):
+    """This class enables the neo4j operator to execute queries against a configured neo4j server.
+    It requires the configuration name as set in Airflow -> Connections ->
+    :param n4j_conn_id:
+    :type str:
+    """
+    n4j_conn_id: str
```

Comment:
I defined this class variable as it was set when the class was instantiated via __init__(). It needs to be held in the class object so it can be consumed later.

If there is a better way to store / retrieve this value, I am happy to change this.

Reply:
@feluelle Waiting for your advice on this please. Where would you have defined this ?